package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "mini-lightbox",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "description": "Minimalist image lightbox",
-  "main": "index.js",
+  "main": "lib/mini-lightbox.js",
```

Comment:
I usually keep the original source in `lib/index.js` and `dist-it` will create the min file in the dist dir: `dist/mini-lightbox.min.js`.

So, it should be fine. :sparkle: